var/spack/repos/builtin/packages/llvm/package.py

```diff
@@ -266,6 +280,12 @@ class Llvm(CMakePackage, CudaPackage):
     depends_on("gmp", when="@:3.6 +polly")
     depends_on("isl", when="@:3.6 +polly")
 
+    conflicts("+flang", when="@:10")
+    conflicts("+flang", when="~clang")
+    conflicts("+lldb", when="~clang")
+    conflicts("+z3", when="@:7")
+    conflicts("+z3", when="~clang")
```

Comment:
Ah, got it, I can resolve the other comments, but I would greatly prefer that these be grouped with the appropriate `variant()` directives so they're easier to take in as a whole.

Reply:
I agree. It's just for simple packages we usually have `variant`s, then `depends_on`s and then `conflicts`, which is not necessary in general, as far as I know, and makes the code significantly less readable in this case.